docker/mysql/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mysql:5.7
+
+COPY mysqld.cnf /etc/mysql/mysql.conf.d/mysqld.cnf
+COPY ./setup.sh /docker-entrypoint-initdb.d/setup.sh
```

Comment:
Not that it matters here but sounds like `ADD` is preferred over `COPY` according to this https://stackoverflow.com/a/24958548/7564563. It lets you do more stuff.

Reply:
`COPY` is more explicit. `ADD` does have some nice features though.

https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy